packages/components/src/templates/next/components/internal/CollectionCard/CollectionCard.tsx

```diff
@@ -30,15 +30,15 @@ const ArticleTextComponent = ({
       <div className={`flex flex-col gap-3`}>
         <div className="flex flex-col gap-1 sm:flex-row sm:gap-2">
           <p className="text-caption-01 text-content-medium">{lastUpdated}</p>
-          <p className="text-caption-01 hidden text-content-medium sm:block">
+          <p className="text-caption-01 text-content-medium hidden sm:block">
```

Comment:
hmm wait what why it this hidden ah?

Reply:
linter! was previously hidden

but it's hidden on mobile cos the category and date collapses into 2 rows